lib/mods/helpers.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
What's the difference between originals-helper.js and helpers.js?

Reply:
Very little it turns out, I could easily put formatReplacement into `originals-helper.js` and I think I'm gonna go ahead and do that